src/ngamsServer/ngamsServer/ngamsServer.py

```diff
@@ -436,6 +435,68 @@ def proxy_request(self, host_id, host, port, timeout=300):
 
         self.send_data(data, mime_type, code=code, hdrs=hdrs)
 
+    def remote_proxy_request(self, request, host, port, timeout=300):
+        """Proxy the current request to remote host ``host``:``port``"""
+
+        url = 'http://{0}:{1}/RETRIEVE'.format(host, port)
```

Comment:
Could `RETRIEVE` (i.e., the path) be a parameter instead of being hardcoded at this level? That would make this more reusable in other situations. It would also make this method look again a bit more similar to `proxy_request`.

Reply:
OK. I will do that